modules/server/src/main/scala/navigate/server/NavigateEngine.scala

```diff
@@ -838,8 +839,12 @@ object NavigateEngine {
     result match {
       case Right(ApplyCommandResult.Paused)    => CommandResult.CommandPaused
       case Right(ApplyCommandResult.Completed) => CommandResult.CommandSuccess
+      case Left(e: TimeoutException)           =>
+        CommandResult.CommandFailure(s"Command timed out after ${e.getLocalizedMessage}")
```

Comment:
The message when the timeout happened was simple "5 seconds". Not very informative.